src/framework/preferences/js/Enactors.js

```diff
@@ -405,20 +448,31 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
         return fluid.roundToDecimal(parseFloat(lineHeight) / fontSize, 2);
```

Comment:
Note that the bug referenced on line 444 applied only to old versions of IE and was closed 11 years ago - we should be able to axe the work-around now

Reply:
There's also another reference to the expired FLUID-4500 on line 433

Reply:
I've removed those and updated the tests.